apps/docs/components/hooks/usePagination.md

```diff
@@ -0,0 +1,334 @@
+        >
+          <button
+            type="button"
+            class="px-4 py-3 rounded-md text-neutral-500 hover:bg-primary-100 hover:text-primary-800 active:bg-primary-200 active:text-primary-900"
+            :aria-current="totalPages === selectedPage"
+            @click="onPageChange(totalPages)"
+          >
+            {{ totalPages }}
+          </button>
+        </div>
+      </li>
+    </ul>
+    <SfButton
+      type="button"
+      aria-label="Go to next page"
+      :disabled="selectedPage >= totalPages"
+      variant="tertiary"
+      @click="next"
+    >
+      <template #suffix>
+        <SfIconChevronRight />
+      </template>
+    </SfButton>
+  </nav>
+</template>
+```
+<!-- end vue -->
+
+::::::
+
+::: slot api
+
+## Parameters
+
+| Name        | Type                  | Default value | Description                                  |
+| ---------   | --------------------- | ------------- | -------------------------------------------  |
+| totalItems  | number                |               | total number of items to be be displayed     |
+| currentPage | number                |  1            | current active page                          |
+| pageSize    | number                |  10           | number of items per page                     |
+| maxPages    | number                |  1            | maximum number of pages to display*          |
+
+*there is an additional page displayed when the default number (`1`) is passed. When the current number is `1` then page `2` and the last pages are visible. When the penultimate page is the current one then pages `1` and the last one are visible.
+ 
+
+## Return value
+
+| Name            | Type              | Default value | Description                                        |
+| ---------        | --------------   | ------------- | -------------------------------------------        |
+| totalPages       | number           |               | total number of pages                              |
+| maxVisiblePages  | number           |               | total number of pages                              |
+| itemsPerPage     | number           |               | total number of pages                              |
+| pages            | number[]         |               | array of displayed pages                           |
+| selectedPage     | number           |               | current page                                       |
+| endPage          | number           |               | number of the last page from displayed pages       |
+| startPage        | number           |               | number of the first page from displayed pages      |
+| next             | function         |               | triggers jump to next page                         |
+| prev             | function         |               | triggers jump to previous page                     |
+| onPageChange     | function         |               | triggers jump to page number passed as parameter   |
```

Comment:
Could you please wrap type so it would be displayed as code as well here ? :)

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2535/commits/3ac7e43a8ab6b69cd07bd650bc5f0163b76f477b